main.go

```diff
@@ -240,6 +250,9 @@ This is equivalent to setting PULUMI_MISSING_DOCS_ERROR=${! VALUE}.`)
 	cmd.PersistentFlags().StringVar(&context.JavaVersion, "java-version", "",
 		"The version of pulumi-java-gen to target.")
 
+	cmd.PersistentFlags().StringVar(&targetBridgeRef, "target-bridge-version", "latest",
+		`The desired bridge version to upgrade to. Git hash references permitted. Defaults to "latest".`)
```

Comment:
Looks like "defaults to latest" can be omitted because the framework prints it anyway.